### PR TITLE
Seperated unused scrape function

### PR DESCRIPTION
**Pull request by Mx772**
Moved into seperate Scrape-example file for use by anyone attempting to scrape instead of using an API.

### CODE REVIEW

File: examples/scrape-example.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+// Using the getData() function found in cronjob/scrape.js 
+// you can convert this into a scrape function vs a API function using the code found below:
```
Comment (aharbis) [inline examples/scrape-example.js:2]: If this is example (?) code, it doesn't need to be commented out does it?

**Review by aharbis — CHANGES_REQUESTED** [review]
What's the purpose of this move, or the code being moved?

**Comment by Mx772** [comment]
It was basically for if the main site takes down their API we have working scrape code. Will uncomment and keep it there. 

Also for other users to see how to scrape.

**Comment by Mx772** [comment]
Uncommented now.

File: examples/scrape-example.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+// Definiting Return-object
+    let status = {
```
Comment (aharbis) [inline examples/scrape-example.js]: All of this code can be left indented.

**Review by aharbis — APPROVED** [review]

**Pull request merged**